### DISCUSSION TITLE
Workflow symbols are not rendered in my editor

### DISCUSSION THREAD

**Discussion by return42** [searx/searx#2403] [Development]
In the workflow integration file are very special symbols which can not be rendered in non graphical text editors ..

https://github.com/searx/searx/blob/14c7cc0e118f1d0873b32b34793cdec2c5c9c13e/.github/workflows/integration.yml#L14

for what are these symbols good?

**Comment by dalf** [comment]
IMO, it makes the CI output more clear
https://github.com/searx/searx/runs/1579393874?check_suite_focus=true

But clearly nothing mandatory.

**Comment by return42** [comment]
> IMO, it makes the CI output more clear

What editor and font do you use?

I am using emacs and the default system fonts (on Ubuntu it is a ubuntu font), but most often (when I'am on my WEB host) I use the emacs in a simple terminal.

**Comment by dalf** [comment]
I use VSCode with different extensions (pylint, etc...).

I can remove them.

**Comment by return42** [comment]
> I can remove them.

Thanks! .. as long as we in programming we should stay with ascii, I think.

**Comment by return42** [comment]
@dalf Do you know how I can _close_ or mark as _resolved_ this discussion?

I found this description: https://docs.github.com/en/free-pro-team@latest/discussions/managing-discussions-for-your-community/moderating-discussions#marking-a-comment-as-an-answer

But I don't have such a  checkmark icon in my github UI to click.

**Comment by unixfox** [comment]
No need to close the discussion because this category is just replies, this discussions category is not only about solving problems, it could just be some discussions about what do to develop for Searx.

**Reply by return42** [reply]
@unixfox thanks for your feedback ..

> this discussions category is not only about solving problems, it could just be some discussions about what do to develop for Searx.

This kind of discussion forum is new to me, so please forgive my dump questions .. I have seen that there are categories which have "answers" and I do not understand why not all categories do have this feature .. I mean, a discussion often starts with a question and when the question is answered why not mark the _solving_ answer?

**Reply by unixfox** [reply]
Well your new GitHub discussion confirm the case where it's not really only about asking a question so there is no answer to mark as solved: https://github.com/searx/searx/discussions/2430

**Reply by return42** [reply]
:) right .. I learn ..
